src/Type/Php/SubstrDynamicReturnTypeExtension.php

```diff
@@ -25,7 +26,7 @@ class SubstrDynamicReturnTypeExtension implements DynamicFunctionReturnTypeExten
 
 	public function isFunctionSupported(FunctionReflection $functionReflection): bool
 	{
-		return $functionReflection->getName() === 'substr';
+		return in_array($functionReflection->getName(), ['substr', 'mb_substr'], true);
```

Comment:
The `substr()` calls below should be changed to use the correct function as well.

Reply:
great catch, thx.